backend/pkg/database/database.go

```diff
@@ -314,7 +315,18 @@ GROUP BY vulnerabilities.id,
          p.name,
          p.version,
          p.id,
-         pv.fix_version
+         pv.fix_version;
+  `
+
+	cisDockerBenchmarkResultsViewQuery = `
+CREATE VIEW cis_d_b_checks_view AS
+SELECT resource_cis_d_b_checks.resource_id,
```

Comment:
`resource_cis_d_b_checks.resource_id AS resource_id`

Reply:
It will be resource_id if the `AS` is not defined. But I agree with you because it is more evident. THX